### ISSUE TITLE
Error 54113 trying to login to paramountplus.com (Qt5 cause I need drm support)

### ISSUE THREAD

**Issue by puppinoo** [qutebrowser/qutebrowser#8363]
**Version info**:
Version info
Yank pastebin URL for version info
         ______     ,,
    ,.-"`      | ,-` |
  .^           ||    |
 /    ,-*^|    ||    |
;    /    |    ||    ;-*```^*.
;   ;     |    |;,-*`         \
|   |     |  ,-*`    ,-"""\    \
|    \   ,-"`    ,-^`|     \    |
 \    `^^    ,-;|    |     ;    |
  *;     ,-*`  ||    |     /   ;;
    `^^`` |    ||    |   ,^    /
          |    ||    `^^`    ,^
          |  _,"|        _,-"
          -*`   ****"""``

qutebrowser v3.3.1
Git commit: 1127c0de1 on main (2024-10-29 11:17:56 +1300)
Backend: QtWebEngine 5.15.15
  based on Chromium 87.0.4280.144
  (source: UA)
Qt: 5.15.10

CPython: 3.11.6
PyQt: 5.15.9

Qt wrapper: PyQt5 (via --qt-wrapper)

colorama: 0.4.6
jinja2: 3.1.4
pygments: 2.18.0
yaml: 6.0.2
adblock: 0.6.0
objc: no
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.6
PyQt5.QtWebKitWidgets: no
PyQt5.sip: 6.7.10
pdf.js: no
sqlite: 3.42.0
QtNetwork SSL: OpenSSL 3.0.10 1 Aug 2023

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Mesa, 4.5 (Compatibility Profile) Mesa 23.2.1-1ubuntu3.1
Platform: Linux-6.5.0-44-generic-x86_64-with-glibc2.38, 64bit
Linux distribution: Ubuntu 23.10 (ubuntu)
Frozen: False
Imported from /usr/local/lib/python3.11/dist-packages/qutebrowser-3.3.1-py3.11.egg/qutebrowser
Using Python from /usr/bin/python3.11
Qt library executable path: /usr/lib/x86_64-linux-gnu/qt5/libexec, data path: /usr/share/qt5

Paths:
cache: /tmp/qutebrowser-basedir-j8xp48p2/cache
config: /tmp/qutebrowser-basedir-j8xp48p2/config
data: /tmp/qutebrowser-basedir-j8xp48p2/data
runtime: /tmp/qutebrowser-basedir-j8xp48p2/runtime

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:00:07

**Does the bug happen if you start with `--temp-basedir`?**:
Yes
**Description**

**How to reproduce**
open from bash this and try to login
env QT_SCALE_FACTOR=1 env QT_QUICK_BACKEND=software /usr/local/bin/qutebrowser --temp-basedir ":open 
https://paramountplus.com" --qt-flag no-sandbox --qt-arg geometry 1280x720+0+ --qt-wrapper PyQt5

P.s.: Can I install Qt6 and still use DRM?

Thanks.

**Comment by The-Compiler** [comment]
I'm assuming I need an account to reproduce this, which I don't have. In any case, this sounds like the website just not supporting such an old Qt/Chromium, and there's likely nothing that qutebrowser can do about that. DRM should work on Qt 6 as well just fine, if you have an appropriate version of Widevine installed.

**Comment by puppinoo** [comment]
Hi and thanks for reply.
You dont need an account. The error appears just clicking on the "Log in" button.
I tried QT6. Login works fine but any drm video stream doesnt starts.
My widevine setup is this:
I have this into config.py
c.qt.args = ['widevine-path=/opt/widevine/libwidevinecdm.so']
(the path contains a recent .so


Thanks.

**Comment by The-Compiler** [comment]
Ah, reproduced, maybe I wasn't correctly running on Qt 5 before. Yep, that's just the server deciding it doesn't like your setup - not qutebrowser's fault. 

Maybe you can make it work by lying about your `content.headers.user_agent` (ideally with an URL pattern and not globally), but you should really find a way to upgrade.

https://bitmovin.com/demos/drm works fine for me on Qt 6.8 and without any path overrides. It looks like it's loading `~/.config/chromium/WidevineCdm/4.10.2830.0/_platform_specific/linux_x64/libwidevinecdm.so` which was probably downloaded by Chromium.

**Comment by puppinoo** [comment]
Thanks for trying.
I switched to Qt6 and I'm trying to launch any DRM protected resource but it gives black screen on Paramount Plus. The thing is that with that exact setup, DRM ParamountPlus works with other browser.
I tried to launch with --temp-basedir and deleted 
I deleted ~/.local/share/qutebrowser and ~/.cache/qutebrowser.
Still black screen.
Youtube works.

https://bitmovin.com/demos/drm doesnt work. It says "NO DRM".

How can I debug?
With older qutebrowser + Qt5 it worked fine (but it's been a long time ago).

Thanks
